README.md

```diff
@@ -37,35 +36,34 @@ exchange traded products.  These include:
 * Warrants
```

Comment:
The contract list is repeated in the #Create a contract section. Do we want it twice?

Reply:
I was just trying to be explicit.  The top is for selling.  The bottom is for figuring out how to use it.